crates/solver/src/arguments.rs

```diff
@@ -141,10 +141,14 @@ pub struct Arguments {
     #[clap(long, env, default_value = "10")]
     pub zeroex_slippage_bps: u32,
 
-    /// The slippage tolerance we apply to the price quoted by oneInchSolver
+    /// The default slippage tolerance we apply to the price quoted by OneInchSolver
     #[clap(long, env, default_value = "10")]
     pub oneinch_slippage_bps: u32,
 
+    /// The maximum slippage in ETH we are willing to incur per trade on 1Inch
+    #[clap(long, env, default_value = "0.07")]
```

Comment:
I think this default needs to be different per network.

Reply:
Good catch! Indeed, this is supposed to be used only on mainnet. For the rest, it should be ignored. In my opinion, this is quite pressing and would be good to move to prod tomorrow with the weekly release, but I still have a hard time navigating in the code, so in case someone has time to fix this, it would be great. Thanks!

Reply:
I'll do it.

Reply:
Thanks @vkgnosis !

Reply:
For future reference, the PR addressing this is: https://github.com/cowprotocol/services/pull/386